wiki/en/Include-Server-Commands.md

```diff
@@ -1,19 +1,19 @@
-- `-d`  or `--discononquit`   Disconnect all clients on quit. Normally, when a server is stopped or restarted, any clients that have not used their "Disconnect" buttons will re-establish connection when the server comes back up again. Using this option forces clients to manually re-establish their connections to the server.  
-- `-e`  or `--directoryserver`  Register the server on a directory (e.g. to set its genre (see also `-o`)). See [server types](#server-types) for further information.
-- `--directoryfile` Enable directory to remember registered servers even if the directory is restarted. You'll need to specify a path to a writable file location in which server information is stored.
-- `-f`  or `--listfilter`     Whitelist servers registering on the server list, format `ip address 1[;ip address 2]` Directory Servers only. See [server types](#server-types)  
+- `-d`  or `--discononquit`   Disconnect all clients on quit. Normally, when a Server is stopped or restarted, any clients that have not used their "Disconnect" buttons will re-establish connection when the Server comes back up again. Using this option forces clients to manually re-establish their connections to the Server.  
+- `-e`  or `--directoryserver`  Register the Server on a Directory (e.g. to set its genre (see also `-o`)). See [Server Types](#server-types) for further information.
+- `--directoryfile` Remember registered Servers even if the Directory is restarted. Directory Servers only. See [this guide](Custom-Directories) for further information. 
+- `-f`  or `--listfilter`     Whitelist Servers registering on the Server list, format `ip address 1[;ip address 2]` Directory Servers only. 
```

Comment:
I tend not to think "server list" should have the capitalisation as it's not _directly_ referencing a running Jamulus instance - just the listing.  Not really fussed, though.

You'll need to do the same capitalisation for "Client", though.  Having "client and Server" all over the place looks odd.

Reply:
Well, I think going lower case might cause more problems than it solves if people start wondering if that means they're filtering things other than Jamulus or something.